apps/src/lib/config/genesis.rs

```diff
@@ -233,6 +232,16 @@ pub mod genesis_config {
         // Hashes of whitelisted txs array. `None` value or an empty array
         // disables whitelisting.
         pub tx_whitelist: Option<Vec<String>>,
+        /// Expected number of epochs per year
+        pub epochs_per_year: u64,
```

Comment:
I think since we already have `min_duration`, we could either use that to estimate `epochs_per_year` (or vise versa)

Reply:
Maybe vice-versa? Since I'm pretty sure we are targeting 365 epochs per year (discussed at retreat).

Reply:
vice-versa sounds good!

Reply:
this is still open right? Let's drop `min_duration` from genesis config and instead set it from something like `let min_duration = 60 * 60 * 24 * 365 / epochs_per_year`